cetmix_tower_server/tests/test_variable_option.py

```diff
@@ -0,0 +1,76 @@
+from odoo.tests import TransactionCase
+
+
+class TestTowerVariableOption(TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.variable = self.env["cx.tower.variable"].create(
+            {
+                "name": "odoo_versions",
+            }
+        )
+
+    def test_create_variable_option(self):
+        """Test creation of a TowerVariableOption record."""
+        option = self.env["cx.tower.variable.option"].create(
+            {
+                "name": "14.0",
+                "variable_id": self.variable.id,
+            }
+        )
+        self.assertEqual(option.name, "14.0")
```

Comment:
You are not overriding 'create' method, so no need to test the core ORM methods.

Reply:
I will fix it

Reply:
But it is not fixed...)